app/src/androidTest/java/org/simple/clinic/TestData.kt

```diff
@@ -777,4 +780,45 @@ class TestData @Inject constructor(
         type = type
     )
   }
+
+  fun encounter(
+      uuid: UUID = UUID.randomUUID(),
+      patientUuid: UUID = UUID.randomUUID(),
```

Comment:
Can we provide explicit `UUID` instead of generating them randomly. It becomes really hard to debug when the test fail. Considering that this is not an integration test. Can we remove the default arguments for `uuid` and `patientUuid`?

Reply:
These are just default values. The whole idea of creating these utility methods is to provide an easy way to create multiple encounters in tests. The caller of this method can provide explicit UUIDs. This will be used when we actually start using `Encounter` in our app.

Reply:
Yes, and that is causing problems because it is hard to track when tests fail. That is why we need to move away from this pattern. There are some places where we could make exceptions. For instance, writing integration tests. Otherwise, we should stop using this pattern.

Reply:
Ok. I will make this change.